src/coordinator/index.js

```diff
@@ -58,6 +67,69 @@ class Coordinator {
     this.edgeNodes = new EdgeNodes(sdk, request, baseUrl);
   }
 
+  /**
+   * Creates an application to a user's favorited applications
```

Comment:
Perhaps I'm reading this incorrectly but this wording seems weird to me. Is this actually creating an application or is this just marking an application as a favorite of a specific user?

What about something like `Adds an application to a user's list of favorited applications`?

Reply:
👍